src/Sarif/Visitors/SarifTransformerUtilities.cs

```diff
@@ -313,5 +323,117 @@ public static AnnotatedCodeLocationImportanceVersionOne CreateAnnotatedCodeLocat
                     return AnnotatedCodeLocationImportanceVersionOne.Important;
             }
         }
+
+        public interface IBlameHunk
```

Comment:
>IBlameHunk [](http://example.com/codeflow?start=25&length=10)

move this and the class  below to it's own file.
you can save those inside sarif folder, just like IHttpClient interface

Reply:
Fixed.